app/views/investigations/_new.html.erb

```diff
@@ -1,5 +0,0 @@
-<h1>New <%= t('investigation') %></h1>
-
-<%= form_for @investigation do |f| %>
-    <%= render :partial => "investigations/form", :locals => { :f => f, :action=>:new , :object => @investigation } -%>
-<% end -%>
```

Comment:
I'm unclear what this has been replaced with, and doesn't it just leave the file blank?

Reply:
I thought these **_new** partials are from when the **study**/**investigation** create modals were shown on the single page. Moreover, I expected these files to be deleted...

Reply:
OK, you're right, I didn't notice they are partials and they had been deleted.